doc/schema/attribute-types/text.md

```diff
@@ -13,43 +13,69 @@ export const Todo: PartialSchema = {
 
 ## Parameters
 
-- `default` [{String}] - The default value of the attribute.  Example: `text({default: "hello"})`
-- `required` [{Boolean=false}] - If the attribute must be provided.
+|      key      | description                                                                 | type            | optional  | default |
+| ------------- | --------------------------------------------------------------------------- | :-------------: | :-------: | :-----: |
+| `default`     | The default value of the attribute. <br/> Example: `text({default: true})`  | `Boolean`       | Yes       |         |
+| `required`    | If the attribute must be provided.  <br/> Example: `text({required: true})` | `Boolean`       | Yes       | `false` |
 
-## Form Controls
+## Database and Sequelize Behavior
 
-`text()` will produce a [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) control. 
+The `text` type will create a sequelize [DataTypes.TEXT](https://sequelize.org/docs/v6/core-concepts/model-basics/#strings) column.
 
+## Middleware Behavior
 
-<details>
-<summary>
+### Querying Data
 
-## Advanced Details
+For text, use any string value in your queries as follows:
 
-</summary>
+```js
+GET /todos?filter[notes][$eq]=this%20is%20super%20important //all todos with notes that equal "this is super important"
+GET /todos?filter[notes][$eq]=null //all todos with notes that are null
+GET /todos?filter[notes][$ilike]=%25important //all todos with notes that end in "important"
+GET /todos?filter[notes][$ilike]=this%25 //all todos with notes that start with "this"
+GET /todos?filter[notes][$ilike]=%25is%20super%25 //all todos with notes that contain "is super"
+```
+
+Note:
+`%25` is the ASCII value for `%`, and serves as a wildcard in `ilike` queries
+`%20` is the ASCII value for space.
+
+### Data Response
 
-### Control Type
+Text data will be returned as a string:
 
 ```js
 {
-  type: "String",
-  allowNull: Boolean, 
-  max: Infinity,
-  min: 0
+  data: {
+    ...
+    attributes: {
+      ...
+      notes: "this is super important",
+    },
+  }
 }
 ```
 
+### Mutating Data
 
-### Sequelize Type
+When creating or updating a text attribute, a string value, or `null` must be provided. Any other value will return a service error.
 
-```js
-{
- type: "TEXT",
- typeArgs: [],
- allowNull: Boolean
-}
+## React Rest Behavior
+Similar to the middleware, you MUST provide react rest models a string value, or `null`. Likewise, they will always return these values:
+
+```ts
+Todo.createOne({ attributes: { notes: "this is super important" } })
+
+const [todo, todoMeta] = hatchedReactRest.Todo.useOne({ id })
+todo.notes //-> string or null
 ```
-  
-</details>
 
+## Grid Behavior
```

Comment:
we don't have anything in the frontend code for this now, but what is the expected behaviour for `reallyrealy really really long` text values? should we do an ellipsis at a certain length? (does mui already handle this?)

we should document expected future behaviour with a 🛑

Reply:
It does not currently handle that. Asking in slack